.github/workflows/auto-label-prs.yml

```diff
@@ -7,14 +7,15 @@ jobs:
   label-new-prs:
     runs-on: ubuntu-latest
     steps:
-      - name: Label new drafts
+      - name: Label drafts
         uses: andymckay/labeler@master
         if: github.event.pull_request.draft == true
         with:
           add-labels: 'A3-inprogress'
-      - name: Label new PRs
+          remove-labels: 'A0-pleasereview'
```

Comment:
We should probably also remove `A2-insubstantial`?

Reply:
I'm not sure. If a PR is a draft, I don't think people can merge it. If there's any logic that will auto-merge things labelled insubstantial that have green CI status, that *should* protect us.

The scenario I want to avoid is:
1. Someone makes a new PR labelled with `insubstantial`
2. They realise they need to change something and mark it as a draft, which auto-removes the `insubstantial` label and adds `inprogress`
3. They make their changes and then set as ready-to-review. The autolabeller will then delete the `inprogress` label and add a `pleasereview` label (since there is no insubstantial label)
4. They then have to manually go and remove the `pleasereview` label manually.

I think it's fine for draft PRs to be incorrectly labelled - this job will make it such that any PRs that are ready for review are correctly labelled.

Reply:
>  If there's any logic that will auto-merge things labelled insubstantial that have green CI status, that should protect us.

We will never have such a thing. We never can merge stuff automatically because of a label.